### PR TITLE
Allow negative performing chains and reduce block packing when the basefee is sky high

### PR DESCRIPTION
**Pull request by vyzo**

### CODE REVIEW

File: chain/messagepool/repub.go [hunk 1]
```diff
@@ -91,7 +98,7 @@ func (mp *MessagePool) republishPendingMessages() error {
 
 		// we don't republish negative performing chains, as they won't be included in
 		// a block anyway
-		if chain.gasPerf < 0 {
+		if !allowNegative && chain.gasPerf < 0 {
```
Comment (Kubuxu) [inline chain/messagepool/repub.go:101]: As a note, we should republish slightly negative chains. Base fee can fall quite quickly.

Comment (Kubuxu) [reply chain/messagepool/repub.go:101]: I didn't even think about this effect. In effect, miners who updated to 0.5.10 won't re-pub messages.

Comment (vyzo) [reply chain/messagepool/repub.go:101]: Not sure about that, we might as well wait for the next republish interval.

Comment (vyzo) [reply chain/messagepool/repub.go:101]: But yeah, the effect is real -- and I didn't even think about it either.

**Comment by vyzo** [comment]
it's a lost cause, we'll have real incentives by uncapping the miner penalty in the next 2 forks.

**Pull request closed without merging**